inc/Plugin.php

```diff
@@ -71,6 +75,10 @@ function ( $project_id, $originals_added, $originals_existing, $originals_obsole
 					return;
 				}
 
+				if ( ! $project->is_active() ) {
```

Comment:
Why not combine it as `if ( ! $project || ! $project->is_active() ) {` as you did above?

Reply:
Did not think of it.